app/graphql/mutations/eras/delete_era.rb

```diff
@@ -0,0 +1,14 @@
+module Mutations
+  module Eras
+    class DeleteEra < ::Mutations::BaseMutation
+      argument :id, ID, required: true
+
+      type Types::EraType
+
+      def resolve(id:)
+        era = Era.find(id)
+        era.delete
+      end
+    end
+  end
+end
```

Comment:
This will go away once my Delete PR is merged

Reply:
You mean like it won't show up as a change anymore because it'll already be merged to main?